### ISSUE TITLE
VSCode OLS crash with euro character

### ISSUE THREAD

**Issue by WalterPlinge** [DanielGavin/ols#15]
### Code snippet
```
package main
import €
main :: proc() {
    i: int;
}
```

### Output
```
c:/Odin/Projects/Test/ols.odin(2:4) illegal character '€': 8364
[Error - 20:49:38] Connection to server is erroring. Shutting down server.
```
Only some types of crash outputs the `illegal character` line.
Every crash outputs that the server is erroring.

### Reproduce (I was experimenting as well)
OLS crashes when a `€` is typed after `import`
OLS won't crash if `€` already exists when OLS is started.
If OLS is started after `€` is typed, typing another `€` crashes it (also with some but not other characters).
If OLS is started after `€` is typed, typing "i: in" in main crashes OLS, as soon as `n` is typed.
OLS doesn't crash when only typing `€` in `main`, only when it's in the `import` statement.

Maybe this is too specific, but that's the behaviour I saw

**Comment by DanielGavin** [comment]
@WalterPlinge 

Can you try and see if the new commit fixed your issue. It did on my end.

**Comment by WalterPlinge** [comment]
@DanielGavin 

Yeah, the new commit fixed it. None of those steps crashed it this time.
